integtest/test_pacman-raw.py

```diff
@@ -1,33 +1,50 @@
 import pytest
 import dfmodules.data_file_checks as data_file_checks
+import dfmodules.integtest_file_gen as integtest_file_gen
 import integrationtest.log_file_checks as log_file_checks
+import integrationtest.config_file_gen as config_file_gen
+
 # Values that help determine the running conditions
 number_of_data_producers=1
 run_duration=60  # seconds
+
 # Default values for validation parameters
 expected_number_of_data_files=1
 check_for_logfile_errors=True
 expected_event_count=run_duration
 expected_event_count_tolerance=2
 
-#wib1_frag_hsi_trig_params={"fragment_type_description": "Pacman", "hdf5_groups": "NDLArTPC/Region000",
-#                           "element_name_prefix": "Element", "element_number_offset": 0,
-#                           "expected_fragment_count": number_of_data_producers,
-#                           "min_size_bytes": 80, "max_size_bytes": 1048656}
-
-wib1_frag_hsi_trig_params={"fragment_type_description": "Pacman",
-                           "hdf5_detector_group": "NDLArTPC", "hdf5_region_prefix": "Region",
-                           "element_name_prefix": "Element", "element_number_offset": 0, "expected_fragment_count": number_of_data_producers,
-                           "min_size_bytes": 80, "max_size_bytes": 1048656}
+wib1_frag_hsi_trig_params={"fragment_type_description": "PACMAN",
+                           "fragment_type": "PACMAN",
+                           "hdf5_source_subsystem": "Detector_Readout",
+                           "hdf5_detector_group": "NDLArTPC", 
+                           "hdf5_region_prefix": "Region",
+                           "element_name_prefix": "Element", 
+                           "element_number_offset": 0, 
+                           "expected_fragment_count": number_of_data_producers,
+                           "min_size_bytes": 80, 
+                           "max_size_bytes": 1048656}
 
 # The next three variable declarations *must* be present as globals in the test
 # file. They're read by the "fixtures" in conftest.py to determine how
 # to run the config generation and nanorc
+
 # The name of the python module for the config generation
 confgen_name="daqconf_multiru_gen"
 # The arguments to pass to the config generator, excluding the json
 # output directory (the test framework handles that)
-confgen_arguments=[ "--host-ru", "localhost", "-o", ".", "-n", str(number_of_data_producers), "--frontend-type", "pacman", "-b", "2500000", "-a", "2500000", "-t", "1.0", "--op-env", "integtest" ]
+detid_ND_LAr = 32 
+number_of_apps = 1 
+hardware_map_contents = integtest_file_gen.generate_hwmap_file( number_of_data_producers, number_of_apps, detid_ND_LAr )
+
+conf_dict = config_file_gen.get_default_config_dict()
+conf_dict["daqconf"]["op_env"] = "integtest"
```

Comment:
This "daqconf" will have to change to "boot" for the latest changes in `daqconf`. I don't think `integrationtest` has been tagged yet, so you'll also have to pull and install that (`git clone git@github.com:DUNE-DAQ/integrationtest.git;pip install -U ./integrationtest`)

Reply:
I tested the change with the `N22-09-13` version. The configuration is successful